playwright/tests/shippingMethods.spec.ts

```diff
@@ -1,14 +1,21 @@
-import { SHIPPING_METHODS } from "@data/e2eTestData";
+import { CHANNELS, SHIPPING_METHODS, WAREHOUSES } from "@data/e2eTestData";
 import { ShippingMethodsPage } from "@pages/shippingMethodsPage";
 import { ShippingRatesPage } from "@pages/shippingRatesPage";
 import { expect, test } from "@playwright/test";
 
 test.use({ storageState: "playwright/.auth/admin.json" });
+let shippingMethodsPage: ShippingMethodsPage;
+let shippingRatesPage: ShippingRatesPage;
+
+
+test.beforeEach(({ page }) => {
+  shippingMethodsPage = new ShippingMethodsPage(page);
+  shippingRatesPage = new ShippingRatesPage(page);
+});
 
 test("TC: SALEOR_31 Create basic shipping method @shipping-method @e2e", async ({
-                                                                                  page,
-                                                                                }) => {
-  const shippingMethodsPage = new ShippingMethodsPage(page);
+  page,
```

Comment:
Do you need to pass` {page}` since it should be passed in beforeeach?

Reply:
fixed